src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -3,35 +3,30 @@ import PropTypes from 'prop-types'
 import moment from 'moment'
 import { translate } from 'react-i18next'
 
-import styles from './LanguageSelection.scss'
+import Dropdown from '../Dropdown/Dropdown'
```

Comment:
create a index.js file like in the other components so you can import `../Dropdown`